backend/api/api.csproj

```diff
@@ -27,4 +27,8 @@
     <PackageReference Include="MQTTnet.Extensions.ManagedClient" Version="4.0.2.221" />
   </ItemGroup>
 
+  <ItemGroup>
+     <InternalsVisibleTo Include="api" />

```

Comment:
What is this for?

Reply:
To make the api internally visible to the test project
https://learn.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-6.0#basic-tests-with-the-default-webapplicationfactory-1